tools/autograd/gen_inplace_or_view_type.py

```diff
@@ -272,12 +275,16 @@ def unpacked_name(arg_name: str) -> str:
 
 
 # e.g. select.int -> select_copy_int_inverse()
-def inverse_view_name(f: NativeFunction) -> str:
-    copy_variant = f"{f.root_name}_copy"
-    overload = f"{f.func.name.overload_name}"
-    if overload != "":
-        overload = "_" + overload
-    return f"{copy_variant}{overload}_inverse"
+def inverse_view_name(f: NativeFunction, g: NativeFunctionsViewGroup) -> str:
+    if g.view_copy is not None:
+        return ViewInverseSignature(g).name()
+    else:
+        # NB: some view ops don't have associated group data (e.g. chunk, narrow)
```

Comment:
hmmm, why do we have the branching in this function?

In my head, we have total control over what the name of the view inverse function is (since the function here isn't even an ATen op, it's a shim that we implement ourselves in `FunctionalInverses.cpp`. So we should just name them whatever makes the codegen easiest to reason about.

Reply:
The branching is because for *reasons* we don't have `NativeFunctionsViewGroup` info for `narrow()` / `chunk()` at least. But you're right that we can do whatever here as long as it matches what's done over in the functionalization codegen. If you're okay with me changing that codegen, I can probably remove the need to pass this info down, simplifying the codegen changes in this PR.

Reply:
yep, changing the kernel names we codegen seems fine if it reduces complexity!